ESMF/UFS/Aerosol_Cap.F90

```diff
@@ -24,7 +24,7 @@ module Aerosol_Cap
   implicit none
 
   ! -- import fields
-  integer, parameter :: importFieldCount = 26
+  integer, parameter :: importFieldCount = 28
```

Comment:
This is probably a good opportunity to use newer syntax that avoids counting by the end user:
```f90
character(len=*), dimension(*), parameter :: importFieldNames =
   ["inst_pres_interface         ", &
    "inst_sensi_heat_flx          ", 
...
```

Reply:
Commit e8425cd827a22cb425f2f7c35d739a1a5f89a41c incorporates the suggested syntax. Thank you!